webui/README.md

```diff
@@ -0,0 +1,29 @@
+# Web UI support files
+This directory contains tests and config files used to validate the HTML output from `reccmp`. When you run `reccmp-reccmp` with the `--html` option, we create a file with the compared entities for the specified target. The page has some interactive elements that allow you to filter entities and inspect the diff if there is one.
+
+The HTML file is created as follows:
+1. Load the contents of `template.html` and `reccmp.js` from `reccmp/assets`.
+2. Insert the contents of `reccmp.js` into a `<script>` element in `template.html`.
+3. Using the completed analysis run, create a serialized report file in JSON format, then insert its contents in a second `<script>` element in `template.html`. The result is that the data is assigned to the `report` variable.
+The `pystache` module is used to combine these elements into the final output file.
+
+## Setup
+The tools in this directory require [node.js](https://nodejs.org/). The `package.json` file lists the dependencies and you can use your package manager of choice to install them. For example: `npm install`.
+
+## Formatting and linting
+[Biome](https://biomejs.dev/) validates the test files and javascript source. You can run it yourself with the command: `npx @biomejs/biome ci <list-of-paths>`.
+
+The CI scripts run from the `webui` directory and check these paths: `npx @biomejs/biome ci e2e ./testdata.json`.
```

Comment:
Now that I look at it again, it would probably be cleaner to add scripts to the package.json for linting and for running the tests, and then executing them via `npm run lint` or `npm run test`, both here and in the CI scripts. We can change this as a drive-by in a later PR, though.